azerothcore/azerothcore-wotlk
# fix(Core/Tools): fix GCC warnings in extractors

### How to test the changes

- Check out this PR and build the core with `-DTOOLS=1`
- Extract client data files (dbc, maps, vmaps, etc..)
- Run the core using the newly extracted data
